docs/servo_power_test/src/operate_port.ts

```diff
@@ -25,14 +25,17 @@ export class OperatePort {
     if (this.port === undefined) return;
     await this.port.open({baudRate: 115200});
   }
-  public async close() {
-    if (this.port === undefined) return;
+  public async readCancel() {
```

Comment:
Ditto, cancelRead(). but this will invoive many other *Cancel() renaming. So this will be done in another commit.

Reply:
+1. Thanks for catching this, Masami!